tests/sav/nitsmells_args1.res

```diff
@@ -0,0 +1,18 @@
+Test ModelBuilder
+*** CODE SMELLS METRICS ***
+--- Code Smells Metrics ---
+-------------------
+Class: Character
+Feature envy :  true
+	Affected method:
+		-quit
+Long method:  Average 1 lines
+	Affected method:
+		-total_strengh has 2 lines
+		-total_endurance has 2 lines
+		-total_intelligence has 2 lines
+-------------------
+Class: Character
+Feature envy :  true
+	Affected method:
+		-hit_points
```

Comment:
Again:

~~~nit
redef fun hit_points do return health
~~~

Here `health` is introduced by `Character` but in another mclassdef.